delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/FormDataExtractor.scala

```diff
@@ -118,23 +127,38 @@ object FormDataExtractor {
         .adaptError {
           case _: EntityStreamSizeException =>
             FileTooLarge(maxFileSize, storageAvailableSpace)
-          case th                           =>
+          case NotARejection(th)            =>
```

Comment:
A rejection can be risen here ?
Not sure if we should just drop this one
Especially to map it to a `MalformedRequestContentRejection`, this may fail for other reasons that do not depend on the client